guides/common/modules/con_troubleshooting-postgresql.adoc

```diff
@@ -0,0 +1,30 @@
+[id="troubleshooting_postgresql_{context}"]
+= Troubleshooting PostgreSQL
+
+If the PostgreSQL 10 module has already been enabled, perform a module reset:
+
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module reset postgresql
+----
+
+If a database was previously created using PostgreSQL 10, perform an upgrade:
+
+. Enable the DNF modules:
++
+[options="nowrap" subs="+quotes,attributes"]
+----
+# {package-manager} module enable {dnf-modules}
```

Comment:
I cannot find `:package-manager:` where I'd expect it. I think you should add it to `attributes-base.adoc`.

& please also re-use it in the ruby file instead of "dnf".

Reply:
Can I add this attribute to the individual files instead of `attributes-base.adoc` ? We still use' yum' in some versions as a package manager. 

(However, I am puzzled how `con_troubleshooting-postgresql.adoc` caught `dnf`. I don't know from where it rendered the attribute value.)

Reply:
Yes, you can add it to `attributes-base.adoc`. Note however that this only works for Foreman on EL8 only. If you want to apply your change to Foreman/Satellite/orcharhino also running on EL7, you'll have to consider this.

Reply:
Then, it will be a good idea to add `:package-manager:` in individual files, instead of `attributes-base.adoc`.  This will allow us to use `yum` for EL7 and other applicable parts.

Reply:
But only for Foreman 3.3 and below. Going forward, there is only "dnf" for Foreman on EL8+. So I'd really appreciate it if you'd create the slightly more complicated solution of adding it _everywhere it's needed for Foreman 3.3 and below only.

Reply:
For Foreman 3.3 and below, I guess the `:package-manager:` has been mentioned in all the applicable files with `dnf` and `yum` values. (For eg., `proc_configuring-repositories.adoc` ). 

Moving forward to `3.4` and so on, we can consider adding `package-manager` directly to `attributes-base.adoc`

I hope, I understood you correctly :smile:

Reply:
Yes, that's my current understanding.

Reply:
Thanks! :+1: 

So, my next step would be to open a separate PR for `master` by adding `package-manager:dnf` to `attributes-base.adoc` and cherry-picking to `3.4` and above. And, remove the `package-manager` from the individual files.

I don't find any attribute-related changes in this PR if I am not wrong.